model/client4.go

```diff
@@ -2124,6 +2128,16 @@ func (c *Client4) CreateCommand(cmd *Command) (*Command, *Response) {
 	}
 }
 
+// UpdateCommand updates a command based on the provided command id string and Command struct
+func (c *Client4) UpdateCommand(commandId string, cmd *Command) (*Command, *Response) {
```

Comment:
sorry, i resubmitted the PR, removing unnecessary comment used for debugging at `command.go`